<html>
<head>
<title>TestNG:  Free CRM App Test Cases</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>Free CRM App Test Cases</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>18/1/3</td>
</tr><tr>
<td>Started on:</td><td>Sat Mar 02 14:08:31 PST 2019</td>
</tr>
<tr><td>Total time:</td><td>207 seconds (207424 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='com.crm.qa.testcases.MathDemo.division()'><b>division</b><br>Test class: com.crm.qa.testcases.MathDemo</td>
<td><div><pre>java.lang.AssertionError: expected [2] but found [3]
	at com.crm.qa.testcases.MathDemo.division(MathDemo.java:20)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace337816703", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace337816703'><pre>java.lang.AssertionError: expected [2] but found [3]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:388)
	at org.testng.Assert.assertEquals(Assert.java:398)
	at com.crm.qa.testcases.MathDemo.division(MathDemo.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>com.crm.qa.testcases.MathDemo@31ef45e3</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='com.crm.qa.testcases.LoginPageTest.loginTest()'><b>loginTest</b><br>Test class: com.crm.qa.testcases.LoginPageTest</td>
<td></td>
<td>5</td>
<td>com.crm.qa.testcases.LoginPageTest@f6c48ac</td></tr>
<tr>
<td title='com.crm.qa.testcases.LoginPageDataDrivenTest.loginToFreeCRM()'><b>loginToFreeCRM</b><br>Test class: com.crm.qa.testcases.LoginPageDataDrivenTest<br>Parameters: qvofreecrm, easypw123</td>
<td></td>
<td>4</td>
<td>com.crm.qa.testcases.LoginPageDataDrivenTest@57fffcd7</td></tr>
<tr>
<td title='com.crm.qa.testcases.LoginPageDataDrivenTest.loginToFreeCRM()'><b>loginToFreeCRM</b><br>Test class: com.crm.qa.testcases.LoginPageDataDrivenTest<br>Parameters: qvofreemarketing, freemarketing</td>
<td></td>
<td>5</td>
<td>com.crm.qa.testcases.LoginPageDataDrivenTest@57fffcd7</td></tr>
<tr>
<td title='com.crm.qa.testcases.MathDemo.sum()'><b>sum</b><br>Test class: com.crm.qa.testcases.MathDemo</td>
<td></td>
<td>0</td>
<td>com.crm.qa.testcases.MathDemo@31ef45e3</td></tr>
<tr>
<td title='com.crm.qa.testcases.HomePageTest.verifyContactsLinkTest()'><b>verifyContactsLinkTest</b><br>Test class: com.crm.qa.testcases.HomePageTest</td>
<td></td>
<td>0</td>
<td>com.crm.qa.testcases.HomePageTest@13deb50e</td></tr>
<tr>
<td title='com.crm.qa.testcases.ContactsPageTest.verifyContactsPageHeader()'><b>verifyContactsPageHeader</b><br>Test class: com.crm.qa.testcases.ContactsPageTest</td>
<td></td>
<td>0</td>
<td>com.crm.qa.testcases.ContactsPageTest@239963d8</td></tr>
<tr>
<td title='com.crm.qa.testcases.ContactsPageTest.verifyCreateNewContacts()'><b>verifyCreateNewContacts</b><br>Test class: com.crm.qa.testcases.ContactsPageTest<br>Parameters: Mr., Tom, Peter, Google</td>
<td></td>
<td>1</td>
<td>com.crm.qa.testcases.ContactsPageTest@239963d8</td></tr>
<tr>
<td title='com.crm.qa.testcases.ContactsPageTest.verifyCreateNewContacts()'><b>verifyCreateNewContacts</b><br>Test class: com.crm.qa.testcases.ContactsPageTest<br>Parameters: Dr., David, Cris, Amazon</td>
<td></td>
<td>0</td>
<td>com.crm.qa.testcases.ContactsPageTest@239963d8</td></tr>
<tr>
<td title='com.crm.qa.testcases.ContactsPageTest.verifyCreateNewContacts()'><b>verifyCreateNewContacts</b><br>Test class: com.crm.qa.testcases.ContactsPageTest<br>Parameters: Mrs., Mukta, Sharma, Ebay</td>
<td></td>
<td>1</td>
<td>com.crm.qa.testcases.ContactsPageTest@239963d8</td></tr>
<tr>
<td title='com.crm.qa.testcases.DealsPageTest.verifyCreateNewDeals()'><b>verifyCreateNewDeals</b><br>Test class: com.crm.qa.testcases.DealsPageTest<br>Parameters: $10 Hair Cut, D-Hairs, Duc Vu, 10.0</td>
<td></td>
<td>2</td>
<td>com.crm.qa.testcases.DealsPageTest@3abbfa04</td></tr>
<tr>
<td title='com.crm.qa.testcases.DealsPageTest.verifyCreateNewDeals()'><b>verifyCreateNewDeals</b><br>Test class: com.crm.qa.testcases.DealsPageTest<br>Parameters: $8 Kid Hair Cut, D-Hairs, Thao Thui, 8.0</td>
<td></td>
<td>2</td>
<td>com.crm.qa.testcases.DealsPageTest@3abbfa04</td></tr>
<tr>
<td title='com.crm.qa.testcases.DealsPageTest.verifyCreateNewDeals()'><b>verifyCreateNewDeals</b><br>Test class: com.crm.qa.testcases.DealsPageTest<br>Parameters: $7 Senior Wednesday, D-Hairs, Duc Vu, 7.0</td>
<td></td>
<td>2</td>
<td>com.crm.qa.testcases.DealsPageTest@3abbfa04</td></tr>
<tr>
<td title='com.crm.qa.testcases.DealsPageTest.verifyCreateNewDeals()'><b>verifyCreateNewDeals</b><br>Test class: com.crm.qa.testcases.DealsPageTest<br>Parameters: $20 for entire family Tuesday, D-Hairs, Duc Vu, 20.0</td>
<td></td>
<td>2</td>
<td>com.crm.qa.testcases.DealsPageTest@3abbfa04</td></tr>
<tr>
<td title='com.crm.qa.testcases.HomePageTest.verifyDealsLinkTest()'><b>verifyDealsLinkTest</b><br>Test class: com.crm.qa.testcases.HomePageTest</td>
<td></td>
<td>0</td>
<td>com.crm.qa.testcases.HomePageTest@13deb50e</td></tr>
<tr>
<td title='com.crm.qa.testcases.DealsPageTest.verifyDealsPageHeader()'><b>verifyDealsPageHeader</b><br>Test class: com.crm.qa.testcases.DealsPageTest</td>
<td></td>
<td>0</td>
<td>com.crm.qa.testcases.DealsPageTest@3abbfa04</td></tr>
<tr>
<td title='com.crm.qa.testcases.HomePageTest.verifyHomePageTitleTest()'><b>verifyHomePageTitleTest</b><br>Test class: com.crm.qa.testcases.HomePageTest</td>
<td></td>
<td>0</td>
<td>com.crm.qa.testcases.HomePageTest@13deb50e</td></tr>
<tr>
<td title='com.crm.qa.testcases.LoginPageTest.verifyLoginPageTitleTest()'><b>verifyLoginPageTitleTest</b><br>Test class: com.crm.qa.testcases.LoginPageTest</td>
<td></td>
<td>0</td>
<td>com.crm.qa.testcases.LoginPageTest@f6c48ac</td></tr>
<tr>
<td title='com.crm.qa.testcases.HomePageTest.verifyUsernameLabelTest()'><b>verifyUsernameLabelTest</b><br>Test class: com.crm.qa.testcases.HomePageTest</td>
<td></td>
<td>0</td>
<td>com.crm.qa.testcases.HomePageTest@13deb50e</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-skipped'>
<tr><td colspan='4' align='center'><b>SKIPPED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='com.crm.qa.testcases.MathDemo.division()'><b>division</b><br>Test class: com.crm.qa.testcases.MathDemo</td>
<td><div><pre>java.lang.AssertionError: expected [2] but found [3]
	at com.crm.qa.testcases.MathDemo.division(MathDemo.java:20)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace2145040140", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace2145040140'><pre>java.lang.AssertionError: expected [2] but found [3]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:388)
	at org.testng.Assert.assertEquals(Assert.java:398)
	at com.crm.qa.testcases.MathDemo.division(MathDemo.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>com.crm.qa.testcases.MathDemo@31ef45e3</td></tr>
<tr>
<td title='com.crm.qa.testcases.MathDemo.division()'><b>division</b><br>Test class: com.crm.qa.testcases.MathDemo</td>
<td><div><pre>java.lang.AssertionError: expected [2] but found [3]
	at com.crm.qa.testcases.MathDemo.division(MathDemo.java:20)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1241182310", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1241182310'><pre>java.lang.AssertionError: expected [2] but found [3]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:388)
	at org.testng.Assert.assertEquals(Assert.java:398)
	at com.crm.qa.testcases.MathDemo.division(MathDemo.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>com.crm.qa.testcases.MathDemo@31ef45e3</td></tr>
<tr>
<td title='com.crm.qa.testcases.MathDemo.division()'><b>division</b><br>Test class: com.crm.qa.testcases.MathDemo</td>
<td><div><pre>java.lang.AssertionError: expected [2] but found [3]
	at com.crm.qa.testcases.MathDemo.division(MathDemo.java:20)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1163664780", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1163664780'><pre>java.lang.AssertionError: expected [2] but found [3]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:388)
	at org.testng.Assert.assertEquals(Assert.java:398)
	at com.crm.qa.testcases.MathDemo.division(MathDemo.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>com.crm.qa.testcases.MathDemo@31ef45e3</td></tr>
</table><p>
</body>
</html>